concepts/type-checking/about.md

```diff
@@ -0,0 +1,144 @@
+# About
+
+Knowning what type an object has is often very important for code to run smoothly and without errors.
+
+Javascript has several ways to check the type of an object.
+
+```exercism/note
+Javascript's type checking mechanisms are always soomewhat unreliable.
+
+For true safety with types, you should probably use TypeScript, a language that builds on JavaScript, but with the type syntax of a static-typed language.
+```
+
+## The `typeof` operator
+
+The `typeof` operator returns the type of its input.
+The output is restricted to one of the [primitive data types][primitives], `"function"` or `"object"`.
```

Comment:
```suggestion
The output is a string matching the name of one of the [primitive data types][primitives], except for `"null"`. 
It can also be `"function"` or `"object"`.
```